client/js/widgets/pile.js

```diff
@@ -23,19 +23,30 @@ class Pile extends Widget {
 
     this.domElement.appendChild(this.handle);
     this.childCount = 0;
+    this.visibleChildren = new Set();
+    this.visibleChildLimit = 5;
```

Comment:
Should this be a property that game designers can adjust?

Reply:
Happy to add this if you think we should. 5 is an arbitrary attempt to ensure that it's unlikely you would notice most of the pile is hidden (in case the widgets in the pile aren't opaque).